### PR TITLE
Enable per crawler thread logging

### PR DESCRIPTION
**Pull request by adrianreber**
MM1 used to log for each crawler the output under <hostid>.log.
This adds the per host logging to the new thread based design.
These log files are referenced on each host page in the web
interface.

### CODE REVIEW

File: utility/mirrormanager2.spec [hunk 1]
```diff
@@ -219,6 +227,8 @@ cp -r utility/zebra-dump-parser $RPM_BUILD_ROOT/%{_datadir}/mirrormanager2/
 
 
 %files crawler
+%config(noreplace) %{_sysconfdir}/logrotate.d/mm2_crawler
+%{_localstatedir}/log/mirrormanager
```
Comment (pypingou) [inline utility/mirrormanager2.spec]: Nit picking but could we add a trailing '/' to indicate it's a folder?

**Comment by pypingou** [comment]
One remark, otherwise, very cool :+1:

**Comment by adrianreber** [comment]
Branch updated with a trailing slash in the spec file.

**Comment by pypingou** [comment]
Awesome :)

**Pull request merged**